src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -8484,6 +8484,36 @@ def update_mc_profile_default(self) -> ManagedCluster:
         mc = self.update_bootstrap_profile(mc)
         return mc
 
+    def update_kubernetes_version_and_orchestrator_version(self, mc: ManagedCluster) -> ManagedCluster:
```

Comment:
Is there any concern about calling it in `update_mc_profile_default `? This way you can save yourself from modifying `custom.py`

Reply:
updated.